Show city bike id when name is not available

jpidelatorre opened the pull request:
Reverted behavior and show citybike id when name is not available. The rest of the behavior appears to be as desired.

closes #485

jpidelatorre committed:
Show city bike id when name is not available

Reverted behavior and show citybike id when name is not available.

andreashelms commented on app/component/map/tile-layer/SelectCityBikeRow.js:
```diff
@@ -26,7 +26,7 @@ function SelectCityBikeRow({ selectRow, name, networks, id }, { config }) {
           </div>
           <span className="choose-row-center-column">
-            <h5 className="choose-row-header">{name}</h5>
+            <h5 className="choose-row-header">{name ?? `Station - ${id}`}</h5>
```
Maybe better use ``name || `Station - ${id}`` if name is empty

jpidelatorre replied:
You are absolutely right. I'm changing it right now.

jpidelatorre committed:
Prevent empty names on shared markers

Using boolean or instead of nullish coalescing would prevent empty titles caused by empty names.

leonardehrenfried commented:
Can you centre the text please?

jpidelatorre committed:
Fix popup alignment #485

Removed additional markup that broke the layout
